stories/components/atoms/NavBarActionButton/NavBarActionButton.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { NavBarActionButton, buttonVariants } from ".";
+import { cn } from "@helpers/frontend-utils";
+
+const meta: Meta<typeof NavBarActionButton> = {
+  title: "Atoms/NavBarActionButton",
+  component: NavBarActionButton,
+  parameters: {
+    nextjs: {
+      appDirectory: true
+    }
+  }
+};
+
+export default meta;
+type Story = StoryObj<typeof NavBarActionButton>;
+
+export const Primary: Story = {
+  args: {
+    children: "NavBarActionButton",
+    className: cn(
+      buttonVariants({ variant: "buttonStyle", size: "md", className: "h-20" })
+    ),
```

Comment:
I've provided an example here of how you can surface the combined Tailwind classes in Storybook so the designers can tweak the button styling.